tools/MakeNiiFilesBIDSCompliant.pl

```diff
@@ -1,133 +1,135 @@
+                       in the JSON as C<Not Supplied>
+-verbose             : if set, be verbose
 
 
 =head1 DESCRIPTION
 
 This **BETA** version script will create a BIDS compliant NIfTI file structure of
 the MINC files currently present in the C<assembly> directory. If the argument
-C<tarchive_id> is specified, only the images from that archive will be
-processed. Otherwise, all files in C<assembly> will be included in the BIDS
-structure, while looping though all the C<tarchive_id>'s in the C<tarchive>
-table.
+C<tarchive_id> is specified, only the images from that archive will be processed.
+Otherwise, all C<tarchive_id>'s present in the C<tarchive> table will be processed.
 
 The script expects the tables C<bids_category> and C<bids_mri_scan_type_rel> to
-be populated and customized as per the project acquisitions. Keep the following
-restrictions/expectations in mind when populating the two database tables.
+be populated and customized as per the project's acquisition protocols. Keep the
+following restrictions/expectations in mind when populating the two database tables.
 
 C<bids_category> will house the different imaging "categories" which a default
 install would set to C<anat>, C<func>, C<dwi>, and C<fmap>. More entries can be
 added as more imaging categories are supported by the BIDS standards.
 
 For the C<bids_mri_scan_type_rel> table, functional modalities such as
-resting-state fMRI and task fMRI expect their BIDSScanTypeSubCategory column be
+resting-state fMRI and task fMRI expect their C<BIDSScanTypeSubCategory> column be
 filled as follows: a hyphen concatenated string, with the first part describing
 the BIDS imaging sub-category, "task" as an example here, and the second
 describing this sub-category, "rest" or "memory" as an example. Note that the
 second part after the hyphen is used in the JSON file for the header "TaskName".
 Multi-echo sequences would be expected to see their C<BIDSMultiEcho> column
 filled with "echo-1", "echo-2", etc...
-
 Filling out these values properly as outlined in this description is mandatory
 as these values will be used to rename the NIfTI file, as per the BIDS
 requirements.
 
 Running this script requires JSON library for Perl.
 Run C<sudo apt-get install libjson-perl> to get it.
 
-=head2 Methods
+
+=head2 METHODS
 
 =cut
 
+
+# TODO: BEFORE RUNNING THE SCRIPT, CONFIGURE TABLES ABOVE + MODIFY CONSTANTS IN THE
+# TODO: SCRIPT WITH THE CONSTANTS BELOW (EXPLAIN WHAT THEY ARE)
+
+
+# Imports
 use strict;
 use warnings;
-use Getopt::Tabular;
 use File::Path qw/ make_path /;
 use File::Basename;
+use Getopt::Tabular;
 use JSON;
-
 use NeuroDB::DBI;
 use NeuroDB::MRI;
 use NeuroDB::ExitCodes;
+use NeuroDB::File;
 
-use NeuroDB::Database;
-use NeuroDB::DatabaseException;
 
-use NeuroDB::objectBroker::ObjectBrokerException;
-use NeuroDB::objectBroker::ConfigOB;
+# Set script's constants here
+my $AUTHORS = ( 'LORIS', 'MCIN', 'MNI', 'McGill University' );
```

Comment:
For now, these are constants that need to be modified by the user at the top of the script. Should it be moved to Configuration? If so, should their be a specific section just for the mnc2bids scripts to avoid confusions?

Reply:
will be done in: https://github.com/aces/Loris/issues/7477